src/gui/plugins/scene3d/Scene3D.cc

```diff
@@ -2252,6 +2318,21 @@ void Scene3D::LoadConfig(const tinyxml2::XMLElement *_pluginElem)
   ignmsg << "View angle service on ["
          << this->dataPtr->viewAngleService << "]" << std::endl;
 
+  // view pose service
```

Comment:
`view pose service` -> `move to pose service`

Reply:
Done in 673ba0fbb1f59383d66ab415a92a768ee1383988